metron-platform/metron-parsers/src/main/java/org/apache/metron/parsers/asa/BasicAsaParser.java

```
@@ -0,0 +1,209 @@
+		    .put("ASA-3-313004", "CISCOFW313001_313004_313008")
+		    .put("ASA-3-313008", "CISCOFW313001_313004_313008")
+		    .put("ASA-4-313005", "CISCOFW313005")
+		    .put("ASA-4-402117", "CISCOFW402117")
+		    .put("ASA-4-402119", "CISCOFW402119")
+		    .put("ASA-4-419001", "CISCOFW419001")
+		    .put("ASA-4-419002", "CISCOFW419002")
+		    .put("ASA-4-500004", "CISCOFW500004")
+		    .put("ASA-6-602303", "CISCOFW602303_602304")
+		    .put("ASA-6-602304", "CISCOFW602303_602304")
+		    .put("ASA-7-710001", "CISCOFW710001_710002_710003_710005_710006")
+		    .put("ASA-7-710002", "CISCOFW710001_710002_710003_710005_710006")
+		    .put("ASA-7-710003", "CISCOFW710001_710002_710003_710005_710006")
+		    .put("ASA-7-710005", "CISCOFW710001_710002_710003_710005_710006")
+		    .put("ASA-7-710006", "CISCOFW710001_710002_710003_710005_710006")
+		    .put("ASA-6-713172", "CISCOFW713172")
+		    .put("ASA-4-733100", "CISCOFW733100")
+		    .put("ASA-6-305012", "CISCOFW305012")
+		    .put("ASA-7-609001", "CISCOFW609001")
+		    .put("ASA-7-609002", "CISCOFW609002")
+            .put("ASA-5-713041", "CISCOFW713041")
+            .build();
+
+    @Override
+    public void configure(Map<String, Object> parserConfig) {
+        String timeZone = (String) parserConfig.get("deviceTimeZone");
+        if (timeZone != null)
+            deviceTimeZone = ZoneId.of(timeZone);
+        else {
+            deviceTimeZone = ZoneOffset.UTC;
+            LOG.warn("[Metron] No device time zone provided; defaulting to UTC");
+        }
+    }
+
+    @Override
+    public void init() {
+        asaGrok = new Grok();
+        InputStream patternStream = this.getClass().getClassLoader().getResourceAsStream("patterns/asa");
+        try {
+            asaGrok.addPatternFromReader(new InputStreamReader(patternStream));
+        } catch (GrokException e) {
+            LOG.error("[Metron] Failed to load grok patterns from jar", e);
```

I think we would want to throw a runtime exception after the LOG.  That is, if the GrokParser itself is a good guide for expected behavior.